### ISSUE TITLE
Feature request: Easynews and packs

### ISSUE THREAD

**Issue by bullishtq** [synclerd/syncler#86]
Please add support for Easynews and also season packs like some Kodi addons allow. Thanks.

love the app but keep having to revert to Kodi just for more options, would love it Syncler offered this instead

**Comment by CiRc2K5** [comment]
Agreed adding EasyNews would offer another Backup option with good Retention

**Comment by synclerd** [comment]
Easynews is added
